src/_locales/de/messages.json

```diff
@@ -100,15 +115,81 @@
   },
 
   "menuRestore": {
-    "message": "Cookies von Datei wiederherstellen",
+    "message": "Cookies aus Datei wiederherstellen",
     "description": "Dropdown menu option."
   },
 
+  "menu_tooltip_dropup_protect": {
+    "message": "Sperre/Entsperre alle Cookies",
+    "description": "Tooltip for the dropup menu."
+  },
+
+  "menu_tooltip_dropup_export": {
+    "message": "Export/Import",
+    "description": "Tooltip for the dropup menu."
+  },
+
   "buttonClose": {
     "message": "Schließen",
     "description": "Close button on modal popovers."
   },
 
+  "buttonContinue": {
+    "message": "Weiter",
+    "description": "Continue button on modal popovers and contextual menus."
+  },
+
+
+
+
+  "contextMenu_domain_copy2Clipboard": {
+    "message": "In Zwischenablage kopieren",
+    "description": "Option for the contextual menu that opens on a domain."
+  },
+
+  "contextMenu_domain_copy2File": {
+    "message": "In Datei speichern",
+    "description": "Option for the contextual menu that opens on a domain."
+  },
+
+  "contextMenu_domain_protect": {
+    "message": "Sperren",
+    "description": "Option for the contextual menu that opens on a domain."
+  },
+
+  "contextMenu_domain_unprotect": {
+    "message": "Entsperren",
+    "description": "Option for the contextual menu that opens on a domain."
+  },
+
+  "contextMenu_domain_delete": {
+    "message": "Löschen",
+    "description": "Option for the contextual menu that opens on a domain."
+  },
+
+  "contextMenu_domain_copy2container": {
+    "message": "In Container kopieren",
+    "description": "Option for the contextual menu that opens on a domain."
+  },
+
+
+
+
+  "container_all": {
+    "message": "All",
+    "description": "Name that matches for all the stores in the browser."
+  },
+
+  "container_default": {
+    "message": "Default",
+    "description": "Name of the default store in the browser."
+  },
+
+  "container_private": {
+    "message": "Private browsing",
```

Comment:
If this is justified in the German version of Firefox, could you translate these 3 sections please? 
(container_all, container_default, container_private)